db/test/test_stats.py

```diff
@@ -360,23 +338,22 @@ def test_get_statistics_history(self):
         self.assertEqual(history[0], {"data": [[1452384000000, 15], [1452470400000, 20]], "name": "Lossless (all)"})
 
     def test_stats_daily(self):
```

Comment:
todo: this test only made sense when we got only daily data even when the database contained hourly information (it checked the one last item at the end). Rewrite to check the cache to get the last item